corehq/apps/reports/standard/cases/case_data.py

```diff
@@ -450,7 +450,8 @@ def case_property_names(request, domain, case_id):
     # We need to look at the export schema in order to remove any case properties that
     # have been deleted from the app. When the data dictionary is fully public, we can use that
     # so that users may deprecate those properties manually
```

Comment:
I don't know much about the data dictionary, but just wondering if a) it is now fully public? and b) if so, does that mean we can avoid `generate_schema_from_builds` entirely?

Reply:
Sort of.  It's fully live on all domains, and it _should_ be a definitive source of case properties.  However, we hamstring its usefulness by making the DD UI only available to projects on a specific subscription plan.  This means other projects can't add properties we don't find automatically (a handy fallback option).

Anyways, I do think the DD and export schemas should have close to the same data in them, and the DD pulls from exports anyways:
https://github.com/dimagi/commcare-hq/blob/b9632b73adbd5b1d7ff98d9c22b79cb5e37c6472/corehq/apps/data_dictionary/util.py#L50-L50

My understanding is that no one has gone to the trouble to:
* Prove that DDs contain all properties defined in export schemas
* Figure out how to move from the export schema to DD without breaking workflows
* Remove or deprecate export schemas